packages/amplify-cli/src/extensions/amplify-helpers/make-id.ts

```diff
@@ -1,11 +1,8 @@
-export function makeId(n) {
-  if (!n) {
-    n = 5;
-  }
+export function makeId(n: number = 5) {
   let text = '';
   const possible = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
 
-  for (let i = 0; i < n; i += 1) {
+  for (let i = 0; i < n; ++i) {
```

Comment:
Will this code face collisions in ID when n is small?

Reply:
Possibly. I just edited this file to fix the types for the time being.